### PR TITLE
[Stats Refresh] Show grey bars when chart is empty

### LINKED ISSUE #11888: [Stats Refresh] Likes graphs do not appear to be working properly.
Steps to reproduce:

1. Open any site that has traffic history.
1. Go to Stats > DWMY and tap "Likes" below the graph.
1. Try changing the selected time period by tapping different parts of the graph or by using the arrows at the top to navigate.

<strong>Result:</strong> graph bars are missing for all likes. ([29s](https://cloudup.com/c8wc7dk-uDs))

chart-bars-missing-for-likes
<sup>Tested with WP Internal 12.6.0.20190604 and WP Internal develop `d780002de0` (12.6 alpha) on iPhone 6S iOS 12.3.1.</sup>

### PR DESCRIPTION
**Pull request by ScoutHarris**
Ref #11876
Fixes #11888 

This change displays light grey bars when a chart view is empty. 

This also fixes an issue where chart Likes was not being updated after the Likes query finished.

To test:

---
Latest Post Summary chart:
- The empty view on this one is a bit hard to trigger.
  - A site with views, but no views in the past 2 weeks.
  - A site with no views, but has Likes or Comments.
- Verify light grey bars appear on the chart.

lps

---
Period Overview chart:
- On a site with no Views, Visitors, Likes, and/or Comments for _all the periods_ on the chart.
- Verify light grey bars appear on the chart.

no_likes

no_views

---
Post Stats chart:
- Access Post Stats for a site with no stats in the past 2 weeks.
- Verify light grey bars appear on the chart.

post_stats

---
Chart Likes:
- On a site with Likes.
- Verify Likes updates after the query finishes.

likes_init

likes_updated

Update release notes:

- [x] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.

### CODE REVIEW

**Comment by ScoutHarris** [comment]
@danielebogo - I'm adding you as a reviewer in case you can get to this before @jklausa . Thanks!

File: WordPress/Classes/ViewRelated/Stats/Charts/PeriodChart.swift [hunk 1]
```diff
@@ -75,6 +75,11 @@ private final class PeriodChartDataTransformer {
             effectiveWidth = range / effectiveBars
         }
 
+        let totalViews = summaryData.compactMap({$0.viewsCount}).reduce(0, +)
+        let totalVisitors = summaryData.compactMap({$0.visitorsCount}).reduce(0, +)
+        let totalLikes = summaryData.compactMap({$0.likesCount}).reduce(0, +)
+        let totalComments = summaryData.compactMap({$0.commentsCount}).reduce(0, +)
```
Comment (jklausa) [inline WordPress/Classes/ViewRelated/Stats/Charts/PeriodChart.swift]: love it!

**Comment by SylvesterWilmott** [comment]
Looks good @ScoutHarris 

Since we cannot apply a gradient to the bars, would you mind including a version here for consideration with full height bars. ie. going to the top line?

**Comment by ScoutHarris** [comment]
Hey @SylvesterWilmott . Honestly I'm not sure. Adding the gradient turned out to be a bit complicated. If we do, it'll be a while. So I went ahead and extended the bar heights with #11971.

**Pull request merged**